packages/editor/src/components/parts/program-start/activity/StartPart.tsx

```diff
@@ -0,0 +1,66 @@
+import { IVY_EXCEPTIONS, IVY_SCRIPT_TYPES, ProgramInterfaceStartData } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../../editors';
+import { ExceptionSelect, PathCollapsible, PathFieldset } from '../../common';
+import { useProgramInterfaceStartData } from './useProgramInterfaceStartData';
+import { ScriptInput, useFieldset } from '../../../../components/widgets';
+
+import { useValidations } from '../../../../context';
+import JavaClassSelector from '../JavaClassSelector';
+
+export function useProgramInterfaceStartPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useProgramInterfaceStartData();
+  const compareData = (data: ProgramInterfaceStartData) => [data.javaClass, data.exceptionHandler, data.timeout];
+  const validation = [...useValidations(['timeout']), ...useValidations(['exceptionHandler']), ...useValidations(['javaClass'])];
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return {
+    name: 'Start',
+    state,
+    reset: { dirty, action: () => reset() },
+    content: <StartPart />
+  };
+}
+
+const StartPart = () => {
+  const { config, update, updateTimeout } = useProgramInterfaceStartData();
+
+  const errorFieldset = useFieldset();
+  const timeoutSeconsFieldset = useFieldset();
+  const timeoutErrorFieldset = useFieldset();
+
+  return (
+    <>
+      <JavaClassSelector config={config.javaClass} onChange={change => update('javaClass', change)} type='ACTIVITY' />
+
+      <PathCollapsible label='Program' path='exceptionHandler' defaultOpen={true}>

```

Comment:
implement default open correctly -> only if the configs are not default stuff

Reply:
Okay, I can do that, but in the already existing component this collapsible is also always open no matter if the value in the error is default or not or if the tab in was closed before.  So if I think about it, no collapsible would be necessary at all?

Reply:
Hmm nice to know. But I still would vote to implement it to be closed if nothing special is configured. 
But ok for me if you leave it as it is for now :)